ios/Podfile.lock

```diff
@@ -630,7 +630,7 @@ PODS:
     - RCT-Folly (= 2021.07.22.00)
     - React-Core
     - ReactCommon/turbomodule/core
-  - RNScreens (3.31.1):
```

Comment:
Is this change required?

Reply:
@LazyAfternoons No, it's not, but... there is no change in `package.json` and after running `yarn && bundle exec pod install` this is what I get. We could just leave it out if we are sure that there won't be any breakage.